cmd/mt-whisper-importer-writer/main.go

```diff
@@ -221,10 +222,17 @@ func (s *Server) chunksHandler(w http.ResponseWriter, req *http.Request) {
 	}
 
 	s.index.AddOrUpdate(mkey, &data.MetricData, partition)
+
+	var wg sync.WaitGroup
+	cb := func() {
+		wg.Done()
+	}
 	for _, cwr := range data.ChunkWriteRequests {
-		cwrWithOrg := cwr.GetChunkWriteRequest(nil, mkey)
+		wg.Add(1)
+		cwrWithOrg := cwr.GetChunkWriteRequest(cb, mkey)
```

Comment:
you dont need to define the `cb` func.  you can just pass wg.Done directly.

```suggestion
		cwrWithOrg := cwr.GetChunkWriteRequest(wg.Done, mkey)
```

Reply:
Though it might be good to keep the `cb` func, if we want to add additional logic, maybe as a starter, a log.Debug

Reply:
Good idea, I'll add a log.Debug and leave the cb func in